code/config.ss

```diff
@@ -1,120 +1,120 @@
 (define track-config
   `((language . "Scheme")
-    
+
     (active . #t)
 
     (blurb . "")
 
     (exercises
-     
+
      ((slug . hello-world)
       (uuid . "f2936612-edc4-44f9-a403-c60f7c6486e4")
       (core . #f)
       ;; nb configlet fmt rejects auto-approve but doesn't mind
       ;; unlocked-by.
-      (auto_approve . #t)
+      (auto-approve . #t)
       (unlocked-by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . hamming)
       (uuid . "ce513303-471b-459a-bcf1-6c69f4d83ae7")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics list))
-     
+
      ((slug . leap)
       (uuid . "b94548df-3b35-4ce1-80c7-36179d2b3b86")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics))
-     
+
      ((slug . grains)
       (uuid . "397d9cfc-43f9-4c57-bc55-5c94ae8f532e")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics math))
-     
+
      ((slug . bob)
       (uuid . "4cf6697c-ad9e-426c-a2d8-6c3695812421")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
       (topics string))
-     
+
      ((slug . raindrops)
       (uuid . "9f13b913-a650-4cbf-a392-4b5614e1aa2a")
       (core . #f)
       (unlocked-by)
       (difficulty . 1)
-      (topics))
-     
+      (topics conditionals association-lists))
```

Comment:
Really none of our exercises should be lacking in topics.  I added these to raindrops, so I could also see the result of rendering a `-` separated topic with the json writer.

Reply:
Hm, `association-lists` feels quite specific... but I'm not against it

Reply:
Agreed, though we should probably worry about that in a separate pull request

Reply:
I have removed that now